coordinator/src/main/scala/filodb.coordinator/DatasetCoordinatorActor.scala

```diff
@@ -342,7 +344,14 @@ private[filodb] class DatasetCoordinatorActor(projection: RichProjection,
       handleFlushErr(t)
 
     case InitIngestion(replyTo) =>
-      activeTable.reloadMemTable()
-      replyTo ! DCAReady
+      try {
+        activeTable.reloadMemTable()
+        replyTo ! DCAReady
+      } catch {
+        case ne: java.nio.file.NoSuchFileException =>
+          // Send DCAReady message When no WAL files exist for this dataset
+          replyTo ! DCAReady
```

Comment:
Might be a good idea to log here, just to let folks know a file was not found and we proceed to initialize (actually what happens?)